src/vstest.console/Resources/Resources.resx

```diff
@@ -734,4 +734,10 @@
   <data name="TestSourcesDiscovered" xml:space="preserve">
     <value>A total of {0} test files matched the specified pattern.</value>
   </data>
+  <data name="TestRunSummaryFailed" xml:space="preserve">
```

Comment:
Why do we need two of these ?

Reply:
Expected string: Passed! Total:{0}  Pass:{1}  Fail:{2}  Skip:{3} Time:{4} ({5})

Reply:
corrected